src/Illuminate/View/ClassLessComponent.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Illuminate\View;
+
+class ClassLessComponent extends Component
```

Comment:
I'm not particularly happy with this class name so any suggestion for a better one is welcome.

Reply:
`AnonymousComponent`?

Reply:
Yeah that's better. Thanks. I'm gonna wait a bit more to see if there are any more suggestions.

Reply:
+1 for AnonymousComponent, seems for language-fitting.

Reply:
Renamed to `AnonymousComponent`. Thanks!